src/j1cli.js

```diff
@@ -203,28 +212,40 @@ async function mutateRelationships(j1Client, relationships, update) {
 }
 
 async function mutateAlertRules(j1Client, rules, update) {
-  const promises = [];
+  const created = [];
+  const updated = [];
+  const skipped = [];
```

Comment:
Are you using these arrays anywhere? I see where you're pushing into them, but I don't see where you're reading them.

Reply:
yes, in the future...

Reply:
ok

Reply:
one more time, pls.  the future is here

Reply:
Ah very nice!